website/integrations/services/owncloud/index.md

```diff
@@ -0,0 +1,223 @@
+            http://127.0.0.1:\d+
+            ```
+- Android Application:
+    - Protocol Settings:
+        - Client ID: use the pre-defined values from [ownCloud](https://doc.owncloud.com/server/latest/admin_manual/configuration/user/oidc/oidc.html#client-id)
+        - Client Secret: use the pre-defined values from [ownCloud](https://doc.owncloud.com/server/latest/admin_manual/configuration/user/oidc/oidc.html#client-secret)
+        - Redirect URI: `oc://android.owncloud.com`
+- iOS Application:
+    - Protocol Settings:
+        - Client ID: use the pre-defined values from [ownCloud](https://doc.owncloud.com/server/latest/admin_manual/configuration/user/oidc/oidc.html#client-id)
+        - Client Secret: use the pre-defined values from [ownCloud](https://doc.owncloud.com/server/latest/admin_manual/configuration/user/oidc/oidc.html#client-secret)
+        - Redirect URI: `oc://ios.owncloud.com`
+
+##### Applications
+
+#### ownCloud
+
+In the authentik Admin Interface, go to **Applications -> Applications**, and create a new Application with the following settings for each of the providers defined above:
+
+- Name: owncloud, owncloud-desktop, owncloud-android, or owncloud-ios
+- Slug: same as name
+- Provider: one of the providers you created in the previous section
+- UI Settings:
+    - Launch URL: You can set this to `blank://blank` to prevent the application from being listed on the authentik
+      home page. This may be useful for the desktop, android and ios applications, since you will not be able
+      to navigate directly to those applications.
+      :::note
+      Sometimes this field glitches out when creating the application, you may need to save the application and then
+      edit it to fill this field.
+      :::
+
+##### Service Discovery
+
+In order for the ownCloud Applications to be able to login via OIDC, your reverse proxy will have to be configured to rewrite
+
+`https://owncloud.company/.well-known/openid-configuration`
+
+to
+
+`https://owncloud.company/index.php/apps/openidconnect/config`
+
+The ownCloud [documentation](https://doc.owncloud.com/server/latest/admin_manual/configuration/user/oidc/oidc.html#set-up-service-discovery) includes an example for accomplishing this with Apache.
+
+In Traefik, use the [replacepathregex](https://doc.traefik.io/traefik/middlewares/http/replacepathregex/) middleware to accomplish the same thing.
+When using Traefik in docker-compose as your reverse proxy, add the following labels to your container. See examples in the docs linked above to accomplish this for other Traefik configurations.
+
+```yaml
+labels:
+  ... # other labels for this service/router
+  traefik.http.routers.owncloud.middlewares: owncloud-oidc-rewrite
+  traefik.http.middlewares.owncloud-oidc-rewrite.replacepathregex.regex: ^/\.well-known/openid-configuration
+  traefik.http.middlewares.owncloud-oidc-rewrite.replacepathregex.replacement: /index.php/apps/openidconnect/config
+```
+
+##### ownCloud OIDC Plugin
```

Comment:
Is this a required step, or optional, or just info about this plug-in? If a required step, let's make the header title more task-specific, like "Install and configure the ownCloud OIDC Plugin" or some such.